### PR TITLE
Fix license headers

### LINKED ISSUE #15116: Update the license headers and the command that do it
According to https://github.com/PrestaShop/PrestaShop/pull/15092#discussion_r313806075 our license headers are wrong, and the automatic command https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Command/UpdateLicensesCommand.php#L40 is wrong too.

We need to
- fix the Command
- fix all the headers 😭

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix license headers and update command
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #15116
| How to test?  | Nothing to test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15426)

### CODE REVIEW

**Comment by matks** [comment]
Here is the list of files for which the diff shows more than 1 add / 1 remove:

```
0	24	admin-dev/themes/new-theme/.webpack/common.js
6	6	admin-dev/themes/new-theme/js/app/cldr/exception/localization.js
6	6	admin-dev/themes/new-theme/js/app/cldr/index.js
6	6	admin-dev/themes/new-theme/js/app/cldr/number-formatter.js
6	6	admin-dev/themes/new-theme/js/app/cldr/number-symbol.js
6	6	admin-dev/themes/new-theme/js/app/cldr/specifications/number.js
6	6	admin-dev/themes/new-theme/js/app/cldr/specifications/price.js
3	3	admin-dev/themes/new-theme/js/pages/error/index.js
0	24	admin-dev/themes/new-theme/tests/cldr/number-formatter.spec.js
0	24	admin-dev/themes/new-theme/tests/cldr/number-symbol.spec.js
0	24	admin-dev/themes/new-theme/tests/cldr/specifications/number.spec.js
0	24	admin-dev/themes/new-theme/tests/cldr/specifications/price.spec.js
3	3	app/Resources/TwigBundle/views/Exception/error.html.twig
3	3	src/Adapter/Configuration/CommandHandler/SwitchDebugModeHandler.php
3	3	src/Core/Domain/Configuration/Command/SwitchDebugModeCommand.php
3	3	src/Core/Domain/Configuration/CommandHandler/SwitchDebugModeHandlerInterface.php
0	25	src/Core/Domain/Theme/Exception/FailedToEnableThemeModuleException.php
5	7	src/PrestaShopBundle/Command/UpdateLicensesCommand.php
3	3	src/PrestaShopBundle/Controller/Admin/ErrorController.php
0	25	src/PrestaShopBundle/Form/Admin/Type/IntegerMinMaxFilterType.php
0	25	src/PrestaShopBundle/Form/Admin/Type/NumberMinMaxFilterType.php
3	3	src/PrestaShopBundle/Resources/views/Admin/Exception/error.html.twig
23	23	src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/form.html.twig
23	23	src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Blocks/footer.html.twig
3	3	src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/showcase_card.html.twig
26	1	tests-legacy/bootstrap.php
0	25	tests/Unit/Adapter/Hook/HookDispatcherTest.php
0	24	tests/Unit/PrestaShopBundle/Twig/Fixtures/modules/template.html.twig
0	24	tests/Unit/Resources/mails/templates/account.html.twig
0	24	themes/classic/templates/catalog/_partials/variant-links.tpl
```

For every other changed file, the only modif is 
` * 2007-2019 PrestaShop and Contributors`
=>
` * 2007-2019 PrestaShop SA and Contributors`

**Comment by matks** [comment]
In order to compute the list, I did:
```
  $ git diff HEAD HEAD^  --numstat > log.txt
  // modifies '1  1  ' to 'aaa' in log.txt
  $ sed '/^aaa/d' log.txt > log2.txt
```

**Comment by matks** [comment]
@Progi1984 Waiting for review 😉

File: admin-dev/themes/new-theme/js/app/cldr/number-formatter.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 /**
- * 2007-2019 PrestaShop.
+ * 2007-2019 PrestaShop SA and Contributors
  *
  * NOTICE OF LICENSE
  *
- * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * This source file is subject to the Open Software License (OSL 3.0)
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/app/cldr/number-formatter.js:6]: @matks Do you really want to change the license ?

Comment (matks) [reply admin-dev/themes/new-theme/js/app/cldr/number-formatter.js:6]: Yes it was a mistake, AFL is for modules while the main project is OSL

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
@PierreRambaud do you think we need QA ? I think not

**Comment by PierreRambaud** [comment]
No need QA for headers license :) Thanks @matks

**Pull request merged**